docs/supported-tools.rst

```diff
@@ -0,0 +1,246 @@
+After it reaches its end of support date,
+we don't offer support for builds/docs using these tools or dependencies,
+you'll need to upgrade in order to receive support.
+For :doc:`/commercial/index` we provide an extended support of six months after the official end of support date.
+
+For tools that define an EOL date, we try to follow that date for our policy.
+For tools that release their versions incrementally without an EOL date,
+we choose a date based on: the release of mayor versions (versions with breaking changes),
+the date since it was last updated, and its usage on our platform.
+
+.. note::
+
+   Some recent versions may be supported, but aren't listed on these tables
+   (specially minor updates from supported versions),
+   contact us if you have doubts.
+
+Default dependencies
+--------------------
+
+We install some dependencies in order to build your project without specifying their dependencies.
+In the past we used to set these dependencies to a specific version and update them after some time,
+but by doing so would break some builds and new projects wouldn't be able to use a new version by default.
+For this reason we are now installing their latest version (or latest supported version) by default.
+
+.. note::
+
+   In order to keep your builds reproducible,
+   it's highly recommended declaring its dependencies and versions explicitly.
+
+   .. TODO: link to this guide once it's written https://github.com/readthedocs/readthedocs.org/issues/7852.
+
+External dependencies (Python)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the dependencies that are installed by default when using a Python environment:
+
+Sphinx:
+  Projects created before Oct 20, 2020 use ``1.8.x``.
+  New projects use the latest version.
+
+Mkdocs:
+  Projects created before April 3, 2019 (April 23, 2019 for :doc:`/commercial/index`) use ``0.17.3``.
+  New projects use the latest version.
+
+sphinx-rtd-theme:
+  Projects created before October 20, 2020 (January 21, 2021 for :doc:`/commercial/index`) use ``0.4.3``.
+  New projects use the latest version.
+
+pip:
+  Latest version by default.
+
+setuptools:
+  Latest version by default.
+
+mock:
+  ``1.0.1`` (could be removed in the future).
+
+pillow:
+  ``5.4.1`` (could be removed in the future).
+
+alabaster:
+  ``0.7.x`` (could be removed in the future).
+
+commonmark:
+  ``0.8.1`` (could be removed in the future).
+
+recommonmark:
+  ``0.5.0`` (could be removed in the future).
+
+External dependencies (Conda)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the dependencies that are installed by default when using a Conda environment:
+
+Conda:
+   Miniconda2 ``4.6.14``
+   (could be updated to use the latest version by default).
+
+Mkdocs:
+  Latest version by default installed via ``conda``.
+
+Sphinx:
+  Latest version by default installed via ``conda``.
+
+sphinx-rtd-theme:
+  Latest version by default installed via ``conda``.
```

Comment:
We may mention these come from `conda-forge` channel.

Reply:
we don't set any explicit channel when adding these deps

Reply:
Oh, we probably should then. The default channel usually has old versions. We will end up with different versions installed if you use `pip` than if you use `conda`.

For example, https://pypi.org/project/sphinx-rtd-theme/ latest version is 0.5.1 and we are currently installing 0.4.3 (https://readthedocs.org/projects/time-test/builds/12859408/)

Reply:
not sure if we control on publishing to conda (either the default or forge), I think the safest way is to install via pip the theme at least.

Reply:
We don't have control on `default` but we do on `conda-forge`.

I'm not sure how safe it is to mix `conda` and `pip` in the same environment or if it's recommended or not. We may need to do some research here before installing a package by default using `pip` in a `conda` environment.

Reply:
@astrojuanlu is it 100% safe to mix conda and pip in the same environment?

Reply:
As far as I understand, no it's not 100 % safe. However, I don't know if we are at 98 %, 99.9 %, or just 100 % and we can stop worrying about this.

I just did this quick test: I tried `mamba install "sphinx<3.5"` on a new conda environment and then `pip install -U sphinx` to see what would happen. Result: pip uninstalled the old Sphinx version (surprising! I think this could not be done in the past, perhaps it changed after https://github.com/conda/conda/issues/7053 was closed), detected that the rest of the dependencies were in place (👍🏽), then pip-installed the newest Sphinx version, and finally it looks like `sphinx-quickstart` just works.

```
$ conda list | grep -i sphinx
sphinx                    3.5.3                    pypi_0    pypi
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    1.0.3                      py_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.4                      py_0    conda-forge
```

I will ask in the conda-forge community.

Reply:
https://matrix.to/#/!dUtrSrxSudChYWSwjL:gitter.im/$XfuNmdhoZhl2SR9_5Xx8189DjIjciL2mITUlUvYQh1I?via=gitter.im&via=matrix.org for future reference

Reply:
It turns out I asked in the wrong room 🤦🏽 https://matrix.to/#/!SOyumkgPRWoXfQYIFH:matrix.org/$1619164399102681JsZJt:matrix.org?via=gitter.im&via=matrix.org